.storybook/package.json

```diff
@@ -18,26 +18,27 @@
     "@babel/core": "^7.22.1",
     "@chromaui/addon-visual-tests": "^1.0.0",
     "@etchteam/storybook-addon-status": "^4.2.4",
-    "@storybook/addon-a11y": "^7.6.17",
-    "@storybook/addon-actions": "^7.6.17",
+    "@storybook/addon-a11y": "^8.0.4",
+    "@storybook/addon-actions": "^8.0.4",
     "@storybook/addon-console": "^3.0.0",
-    "@storybook/addon-designs": "^7.0.9",
-    "@storybook/addon-docs": "^7.6.17",
-    "@storybook/addon-essentials": "^7.6.17",
-    "@storybook/addon-interactions": "^7.6.17",
+    "@storybook/addon-designs": "^8.0.0",
+    "@storybook/addon-docs": "^8.0.4",
+    "@storybook/addon-essentials": "^8.0.4",
+    "@storybook/addon-interactions": "^8.0.4",
+    "@storybook/addons": "^7.6.17",
     "@storybook/api": "^7.6.17",
```

Comment:
That's a good call-out. I'd left them as they were because there were no newer versions for them, but after digging into each of them, they have been deprecated for Storybook 8, and the contents of the packages have been moved to _different_ packages.

I'm going to push up another commit to:
* remove the packages that we'll no longer use
* update some of the `template.js` files that were using hooks from these older packages and point their `import`s to the correct (new) packages.